frontend/javascripts/oxalis/model/accessors/volumetracing_accessor.ts

```diff
@@ -778,3 +780,46 @@ export function getMeshInfoForSegment(
   if (meshesForAddCoords == null) return null;
   return meshesForAddCoords[segmentId];
 }
+
+export function needsLocalHdf5Mapping(state: OxalisState, layerName: string) {
+  const volumeTracing = getVolumeTracingByLayerName(state.tracing, layerName);
+  if (volumeTracing == null) {
+    return false;
+  }
+
+  return (
+    volumeTracing.hasEditableMapping ||
+    state.uiInformation.activeTool === AnnotationToolEnum.PROOFREAD
```

Comment:
no, once an editable mapping exists, the front-end will always take care of the mapping to avoid frequent reloads (as we expect that the user will mostly do proofreading in such an annotation). we might change this logic later at some point, but for now I think this is a good strategy.